File: api-dot-net/Services/CodeExecution/Dto/TestCaseResultDto.cs

namespace CjsApi.Services.CodeExecution.Dto
{



    public class TestCaseResultDto
    {
        public int Index { get; set; }
        public string Input { get; set; } = "";
        public string Output { get; set; } = "";
        public string Expected { get; set; } = "";
        public bool Passed { get; set; }
    }

}

File: java-api/cjs-service/src/main/java/com/cjs/cjs_service/dto/TestCaseResultDto.java
package com.cjs.cjs_service.dto;



import com.fasterxml.jackson.annotation.JsonInclude;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@JsonInclude(JsonInclude.Include.NON_NULL)
public class TestCaseResultDto {

    private int index;
    private String input = "";
    private String output = "";
    private String expected = "";
    private boolean passed;

}

